src/features/movies/MovieList/index.js

```diff
@@ -47,9 +51,26 @@ function MovieList() {
 			</Container>
 		);
 	}
+
+	if (loadingSearch) {
+		return (
+			<Section>
+				<SectionTitle>Search result for "{searchQuery}"</SectionTitle>
```

Comment:
That part renders on each peage reload, even if there was no input in search field. I think we have to use different condition as we can't have:
> Search results for "" 

show every time someone reloads / enters the app

Reply:
yea i know but , I have no idea how to do it yet :/ but i try  fix it:)